package.json

```diff
@@ -14,7 +14,8 @@
     "lint": "fedx-scripts eslint --ext .js --ext .jsx .",
     "snapshot": "fedx-scripts jest --updateSnapshot",
     "start": "fedx-scripts webpack-dev-server --progress",
-    "test": "TZ=UTC fedx-scripts jest --coverage --passWithNoTests"
+    "test": "TZ=UTC fedx-scripts jest --coverage --passWithNoTests",
+    "stubs": "$(find . -name pact-stub-service | grep -e 'bin/pact-stub-service$' | head -n 1) ./pacts/frontend-app-profile-edx-platform.json --port 18000"
```

Comment:
When I try to run this, it's looking for a Ruby installation (which I don't have). Also, I know it's from the official docs, but that find/grep/head pipeline seems like it would randomly find the wrong file:

```
timmc@edx-timmc:~/edx-repos/frontend-app-profile$ find . -name pact-stub-service | grep -e 'bin/pact-stub-service$'
./node_modules/@pact-foundation/pact-core/standalone/linux-x64-1.92.0/pact/lib/vendor/ruby/2.4.0/gems/pact-mock_service-3.9.0/bin/pact-stub-service
./node_modules/@pact-foundation/pact-core/standalone/linux-x64-1.92.0/pact/lib/vendor/ruby/2.4.0/bin/pact-stub-service
./node_modules/@pact-foundation/pact-core/standalone/linux-x64-1.92.0/pact/bin/pact-stub-service
./node_modules/@pact-foundation/pact-core/standalone/darwin-1.92.0/pact/lib/vendor/ruby/2.4.0/gems/pact-mock_service-3.9.0/bin/pact-stub-service
./node_modules/@pact-foundation/pact-core/standalone/darwin-1.92.0/pact/lib/vendor/ruby/2.4.0/bin/pact-stub-service
./node_modules/@pact-foundation/pact-core/standalone/darwin-1.92.0/pact/bin/pact-stub-service
./node_modules/@pact-foundation/pact-core/standalone/win32-1.92.0/pact/lib/vendor/ruby/2.4.0/gems/pact-mock_service-3.9.0/bin/pact-stub-service
./node_modules/@pact-foundation/pact-core/standalone/win32-1.92.0/pact/lib/vendor/ruby/2.4.0/bin/pact-stub-service
./node_modules/.bin/pact-stub-service
```

I don't really understand Node's path management, but when I experimented with the following replacement it seemed to work:

```suggestion
    "stubs": "pact-stub-service ./pacts/frontend-app-profile-edx-platform.json --port 18000"
```

...even though I don't have Ruby installed. I think it's using a Dockerized version when called this way.